### PR TITLE
Rapid7-c2c

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — DISMISSED** [review]
- Minor formatting fixes
- You have the same logo twice - can you remove one instance, and then use the other one throughout the docs?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+## Data Source
+
+Rapid7 source ingests inventory, assessment, and summary details for assets from [Assets API](https://help.rapid7.com/insightvm/en-us/api/integrations.html#operation/searchIntegrationAssets) and vulnerabilities from [Vulnerabilities API](https://help.rapid7.com/insightvm/en-us/api/integrations.html#operation/searchIntegrationVulnerabilities).
+
+## Meta-data Fields
+
+Metadata fields will be set, if the integration is configured with the SIEM forward option. See **Metadata Fields** table below:
+
+| Field Name | Value |
+| --- | --- |
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md:23]:
```suggestion
| :--- | :--- |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+<img src={useBaseUrl('img/send-data/rapid7-logo.png')} alt="rapid7-icon" width="80" />
+
+The Rapid7 source collects asset and vulnerabilities from [Rapid7 InsightVM](https://help.rapid7.com/insightvm/en-us/api/integrations.html) API and sends it to Sumo Logic. InsightVM provides a fully available, scalable, and efficient way to collect vulnerability data and minimize risk. InsightVM automatically evaluates changes in user's networks, allowing security professionals to better understand and quickly manage the risk posed to their organization.
+
+## Data Source
+
+Rapid7 source ingests inventory, assessment, and summary details for assets from [Assets API](https://help.rapid7.com/insightvm/en-us/api/integrations.html#operation/searchIntegrationAssets) and vulnerabilities from [Vulnerabilities API](https://help.rapid7.com/insightvm/en-us/api/integrations.html#operation/searchIntegrationVulnerabilities).
+
+## Meta-data Fields
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md:18]:
```suggestion
## Metadata Fields
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+id: rapid7-source
+title: Rapid7 Source
+sidebar_label: Rapid7 Source
+description: Learn how to collect asset and vulnerabilities from Rapid7 InsightVM
+
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md:6]: extra line

```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+---
+id: rapid7-source
+title: Rapid7 Source
+sidebar_label: Rapid7 Source
+description: Learn how to collect asset and vulnerabilities from Rapid7 InsightVM
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source.md:5]:
```suggestion
description: Learn how to collect assets and vulnerabilities from Rapid7 InsightVM.
```

**Review by JainM6 — APPROVED** [review]
LTGM

**Review by kimsauce — CHANGES_REQUESTED** [review]
It looks like you still have two logos attached. Can you delete one, @JV0812 ?

Screenshot 2023-04-18 at 9 26 34 AM

**Pull request merged**